can_tools/scrapers/base.py

```diff
@@ -274,6 +276,9 @@ def extract_CMU(
         var_name: str
             The name of the column in `df` that should be used to lookup
             items from the `cmu` dict for unpacking columns
+        skip_columns: List[str] (default=[])
+            Can be set instead of ``columns`` if there are a small number
+            of columns that should not be set
```

Comment:
Can you clarify what the contract is between this and `columns`?  The comment suggests you can't use them together, but the code looks like they're used together (you'd only add a column to `skip_columns` if it was already in `columns`?)...

Reply:
Yeah this is a bit hacky.

I use it when I want to preserve the value of one of these 7 columns (category, measurement, unit, age, race, ethnicity, sex), but set the value of the other 6. In this PR, I leave `columns` as the default and then set `skip_column = ["age"]` to make sure the age column is not overwritten in this function.

A bit wonky, I agree